penny_university_frontend/src/tests/Error.test.js

```diff
@@ -0,0 +1,19 @@
+import reducer from '../reducers'
+import fetchMock from 'fetch-mock'
+import {makeMockStore, initialState} from './config'
+import {loadChatsList} from '../actions'
+
+describe('error reducer', () => {
+  fetchMock.get('*', () => {
+    throw new Error('It failed!')
+  })
+
+  it('should put error message in state', () => {
+    const store = makeMockStore(initialState)
+
+    return store.dispatch(loadChatsList('1')).then(() => {
+      const state = reducer(initialState, store.getActions()[1])
+      expect(state.error).toEqual('It failed!')
+    })
+  })
+})
```

Comment:
new line

I found our recently that you can set your IDE to always have a trailing newline upon save.

Reply:
Thank you! For some reason Intellij does it for me automatically but pycharm isn't set up to. I'll do that.